Framework/Algorithms/src/FindCenterOfMassPosition2.cpp

```diff
@@ -114,14 +86,15 @@ double initBoundingBox(WorkspaceBoundingBox &boundingBox, const int numSpec, con
 
 // In subsequent iterations check if spectra fits in the normalized bounding box(generated by previous iterations)
 // If so, update position
-double updateBoundingBox(WorkspaceBoundingBox &boundingBox, WorkspaceBoundingBox previousBoundingBox, const int numSpec,
-                         const double beamRadius, const bool directBeam) {
+double updateBoundingBox(WorkspaceBoundingBox &boundingBox, WorkspaceBoundingBox &previousBoundingBox,
+                         const int numSpec, const double beamRadius, const bool directBeam) {
   double totalCount = 0;
+  const auto spectrumInfo = boundingBox.getWorkspace()->spectrumInfo();
```

Comment:
The `ExperimentInfo::spectrumInfo` method returns a `const&` - can we further avoid a copy with `const auto&` here?

Reply:
so the problem wasn't as much having it in the loop, but copying it in the loop

Reply:
Yep. I'm happy to merge this if we want to fix this later? Just while we were here I thought we might as well avoid as much as possible as their could  be some expensive vectors inthe info object.

Reply:
I'd prefer fixing now, as it's a one character thing.